src/test/java/org/cactoos/TextTest.java

```diff
@@ -37,12 +38,12 @@
 
     @Test(expected = IllegalArgumentException.class)
```

Comment:
@dgroup The same above.

Reply:
@fabriciofx disagree, https://github.com/yegor256/cactoos/pull/990#discussion_r247960573